cms-themes.md

```diff
@@ -107,7 +109,7 @@ The code in the PHP section executes every time before the template is rendered.
     url = "/blog"
     layout = "default"
     ==
-    <?
+    <?php
```

Comment:
Short tag is fine here, it will trigger the syntax highlighting and is stripped out by the CMS when parsed.